foundry.toml

```diff
@@ -13,9 +13,13 @@ solc = "0.5.17"
 legacy = true
 fs_permissions = [
     { access = "read", path = "./broadcast"}, 
-    { access = "read", path = "./script/dependencies.json"}
+    { access = "read", path = "./script/upgrades/dependencies.json"}
 ]
 
+# libraries = [
```

Comment:
Sadly, yes.

There is a bug in Foundry scripting with verifying deployed libraries in sourcify. It might be a very localized issue with how this works on Celo testnets, it might not, but we will learn more as we develop.

Right now the issue presents itself by crashing the script in which a library is deployed, during the verification phase. In our case, it's the `MU01-Create-Implementations.sol` script. The deployment works but the script cannot verify the `AddressLinkedList` library which results in none of the other contracts getting verified on Sourcify.

We solved it temporarily by executing the script once to deploy the library, then configuring it, which tells the script to skip deploying the library and use this instead. By skipping the deployment we also skip verification and thus the script executes correctly and verifies the contracts we care about.

The problem with the configuration is that we can't segment it on which network we're running so I commented it out to prevent us from just running this on a different network and linking to a library that doesn't exist. But I decided to leave it so that we can configure it on future deployments. Will add this to the deployment docs as well as a known issue.

Reply:
good to know, thanks for the context!